vllm/core/scheduler.py

```diff
@@ -364,6 +367,12 @@ def has_unfinished_seqs(self) -> bool:
     def get_num_unfinished_seq_groups(self) -> int:
         return len(self.waiting) + len(self.running) + len(self.swapped)
 
+    def flush_finished_requests_ids(self) -> List[str]:
```

Comment:
nit: suggest `get_and_reset_finished_request_ids` so that the reader does not need to wonder where it is "flushed" to